src/setup.ts

```diff
@@ -29,7 +29,7 @@ const setCurrentBlockchainState = async () => {
     return;
   }
 
-  const client = getInstance();
+  const client = getInitInstance();
```

Comment:
The PR title suggests that this is changing it for projects with missing endpoints, but this looks like it's for _all_ projects? Maybe we could try `getInstance` first, and fallback to this new function?